README.md

```diff
@@ -1,115 +1,40 @@
-# plugin-&lt;REPLACE ME&gt;
+# Salesforce DX
 
-Change above to <REPLACE_ME> before finalizing
+A core plugin to the Salesforce DX CLI that provides all commands under the force topic.
 
-&lt;REPLACE ME DESCRIPTION START&gt;
+- [Salesforce DX Setup Guide](https://sfdc.co/sfdx_setup_guide)
+- [Salesforce DX Developer Guide](https://sfdc.co/sfdx_dev_guide)
+- [Salesforce CLI Command Reference](https://sfdc.co/sfdx_cli_reference)
+- [Salesforce Developer Tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)
 
-This repository provides a template for creating a plugin for the Salesforce CLI. To convert this template to a working plugin:
+## Installing
 
-1. Clone this repo
-2. Delete the .git folder
-3. Replace filler values
-   a) Every instance of `<REPLACE_ME>` can be directly substitued for the name of the new plugin. However beware, things like github paths are for the salesforcecli Github organization
-   b) Search for case-matching `REPLACE` to find other filler values, such as for the plugin description
-4. Use `git init` to set up the desired git information
-5. Follow the getting started steps below until the `sfdx hello:org` commmand is functioning
-6. In order to prevent build failures on the intial build, you will need to do the following:
-   1. `npm publish`
-   2. `git tag v1.0.0 ; git push origin v1.0.0`
+Download and install the [Salesforce DX CLI](https://developer.salesforce.com/tools/sfdxcli).
 
-&lt;REPLACE ME DESCRIPTION END&gt;
-
-### Everything past here is only a suggestion as to what should be in your specific plugin's descsription.
-
-## Getting Started
-
-To use, install the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli) and run the following commands.
+By default, the Salesforce DX plugin is bundled with the cli:
```

Comment:
i.e.

By default, the Salesforce DX plugin is bundled with the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli).